packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -1317,6 +1319,15 @@ export const BarfTurnQuest: Quest<GarboTask> = {
         !(totalTurnsPlayed() % 11) &&
         meatMood().execute(estimatedGarboTurns()),
       post: () => {
+        if (
+          !get("_seadentWaveUsed") &&
+          // Seadent can change names, so we use the item ID to avoid mafia errors
```

Comment:
This doesn't make any sense. Address the item by its canonical name like we would with pantogram or any other name-changing item